portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,155 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Viviana Michel</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <div class="topnav">
+     <a href="#home" style="float:left"> VIVIANA MICHEL</a>
+     <a href="#about" style="float:right">CONTACT</a>
+     <a href="#contact" style="float:right">PROJECTS</a>
+     <a href="#about" style="float:right">RÉSUMÉ</a>
+  </div>
   <body>
+      <div>
+          <form action="/data" method="POST">
+          <input type="name" name="name-choice">
+          <input type="name" name="comment-choice">
+          <input type="submit" />
+          </form>
+          <button onclick="getDataStats()">Past Comments</button>
+          <ul id="data-container"></ul>
+      </div>
+      <img src="images/me.jpg" align=left width="350" height="400">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h2>Hello! My name is Viviana Michel</h2>
+      <p>I am a rising junior studying computer science and data science at UC Berkeley. I grew up in Oxnard, California which is where I have 
+          lived my whole life and where I took my first CS class on a whim. Since then I knew computer science was something I wanted to make a career in.</p>
+          <button onclick="getResponse()">hello</button>
+          <div id="data-container"></div>
+      <br><br><br><br><br><br><br><br>
+    </div>
+    <div id="resume">
+        <h4>RÉSUMÉ</h4>
+        <h4 id="title">EDUCATION</h4><hr>
+        <img src="https://brand.berkeley.edu/wp-content/uploads/2016/10/ucbseal_139_540.png" id="image" align=left width="90" height="90" alt="berkeley logo">
+         <h3>
+            <span id="left">University of California, Berkeley</span>
+            <span id="right">2018-2022</span>
+        </h3>
+        <h4 id="major">Data Science and Computer Science, B.A<h4>
+        <h4 id="clist">Relevant Coursework:</h4>
+            <ul id="clist">
+            <li>Structure and Interpretations of Computer Programs</li>
+            <li>Data Structures</li>
+            <li>Discrete Mathematics and Probability Theory</li>
+            <li>The Foundations of Data Science</li>
+            <li>Designing Information Devices and Systems I</li>
+            </ul><br>
+        <h4 id="title">RELEVANT EXPERIENCE</h4><hr>
+        <h3>
+            <span id="left">Oculate Full-Stack Software Engineer Intern</span>
```

Comment:
id should be unique in a page. For names that you want to associate with multiple elements you should use classes instead.